runtime-light/stdlib/crypto/crypto-functions.cpp

```diff
@@ -7,30 +7,23 @@
 #include "common/tl/constants/common.h"
 #include "runtime-light/stdlib/component/component-api.h"
 #include "runtime-light/tl/tl-core.h"
-
-namespace {
-
-// Crypto-Specific TL magics
-
-constexpr uint32_t TL_CERT_INFO_ITEM_LONG = 0x533f'f89f;
-constexpr uint32_t TL_CERT_INFO_ITEM_STR = 0xc427'feef;
-constexpr uint32_t TL_CERT_INFO_ITEM_DICT = 0x1ea8'a774;
-
-constexpr uint32_t TL_GET_PEM_CERT_INFO = 0xa50c'fd6c;
-constexpr uint32_t TL_GET_CRYPTOSECURE_PSEUDORANDOM_BYTES = 0x2491'b81d;
-} // namespace
+#include "runtime-light/tl/tl-functions.h"
+#include "runtime-light/tl/tl-types.h"
 
 task_t<Optional<string>> f$openssl_random_pseudo_bytes(int64_t length) noexcept {
   if (length <= 0 || length > string::max_size()) {
     co_return false;
   }
 
-  tl::TLBuffer buffer;
-  buffer.store_trivial<uint32_t>(TL_GET_CRYPTOSECURE_PSEUDORANDOM_BYTES);
-  buffer.store_trivial<int32_t>(length);
-
   // TODO think about performance when transferring data
 
+  tl::GetCryptosecurePseudorandomBytes request {
+    .size = static_cast<int32_t>(length)
+  };
+
+  tl::TLBuffer buffer;
+  request.store(buffer);
+
   string request_buf;
```

Comment:
Not a blocker, but I think it's better to reduce number of names we introduce into a scope. For example, here we can just do following:
```
tl::TLBuffer buffer;
request.store(buffer);
auto query = f$component_client_send_request(string{"crypto"}, string{buffer.data(), buffer.size()});
```